Reply/app/src/main/res/layout/email_item_layout.xml

```diff
@@ -31,61 +31,81 @@
         android:id="@+id/email_frame"
         android:layout_height="wrap_content"
         android:layout_width="match_parent"
+        android:layout_marginEnd="@dimen/grid_2"
+        android:layout_marginStart="@dimen/grid_2"
         android:layout_marginVertical="@dimen/grid_0_25"
-        android:layout_marginHorizontal="@dimen/grid_0_5"
-        android:background="?attr/colorSecondary">
+        android:layout_marginHorizontal="@dimen/grid_0_5">
 
         <com.google.android.material.card.MaterialCardView
             android:id="@+id/card_view"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:elevation="@dimen/plane_00"
             android:clickable="true"
             android:focusable="true"
             android:onClick="@{(view) -> listener.onEmailClicked(view, email)}"
             android:onLongClick="@{(view) -> listener.onEmailLongPressed(email)}"
             app:cardPreventCornerOverlap="false"
+            android:outlineAmbientShadowColor="@android:color/transparent"
+            android:outlineSpotShadowColor="@android:color/transparent"
+            style="?attr/materialCardViewElevatedStyle"
```

Comment:
nit: usually a style is set as the first thing on the view. like:

        <com.google.android.material.card.MaterialCardView
            style="?attr/materialCardViewElevatedStyle"
            android:id="@+id/card_view"
            ...